src/cosalib/aws.py

```diff
@@ -188,4 +222,7 @@ def aws_cli(parser):
     parser.add_argument("--public", action="store_true", help="Mark images as publicly available")
     parser.add_argument("--tags", help="list of key=value tags to attach to the AMI",
                         action='append', default=[])
+    parser.add_argument("--winli", action="store_true", help="create an AWS Windows LI Ami")
+    parser.add_argument("--windows-ami", help="Windows Server AMI ID used to create AWS Windows LI image")
+    parser.add_argument("--winli-instance-type", help="ec2 instance type used to create AWS Windows LI image (default: t2.large)")
```

Comment:
```suggestion
    parser.add_argument("--winli-instance-type", help="ec2 instance type used to create AWS Windows LI image")
```

I'd just leave off this detail here as it could become stale.

Reply:
SGTM. That's why I created the option in the first place, so we wouldn't hard code it. Just like the windows server ami, this can become stale. Although, much less frequently.

Reply:
Done!